pkg/flare/config_check_test.go

```diff
@@ -212,6 +212,78 @@ func TestContainerExclusionRulesInfo(t *testing.T) {
 	}
 }
 
+func TestPrintClusterCheckConfig_SecretScrubbing(t *testing.T) {
+	testCases := []struct {
+		name             string
+		config           integration.Config
+		checkName        string
+		shouldNotContain []string // strings that should NOT be present in output
+	}{
+		{
+			name: "JSON format with various secret field names",
+			config: integration.Config{
+				Name: "complex",
+				Instances: []integration.Data{
+					integration.Data(`{"password": "instancepass", "username": "user1"}`),
+					integration.Data(`{"token": "instancetoken", "host": "host2"}`),
+				},
+				InitConfig:   integration.Data(`{"secret": "initsecret", "timeout": 30}`),
+				MetricConfig: integration.Data(`{"authKey": "metricauth", "interval": 60}`),
+				LogsConfig:   integration.Data(`{"privKey": "logpriv", "source": "app"}`),
+				ClusterCheck: true,
+			},
+			shouldNotContain: []string{
+				"instancepass", "instancetoken", "initsecret", "metricauth", "logpriv",
+			},
+		},
+		{
+			name: "YAML with various secret field names",
+			config: integration.Config{
+				Name: "multi-secret",
+				Instances: []integration.Data{
+					integration.Data(`
+database:
+  host: db.example.com
+  username: dbuser
+  password: dbpass123
+  auth_token: token789
+api:
+  endpoint: https://api.example.com
+  api_key: abcdef1234567890abcdef1234567890
```

Comment:
Will this also get scrubbed? I would expect the scrubber to replace all sub the last 5 characters. If so, can we add this value to the `shouldNotContain` list?